sdk/formrecognizer/Azure.AI.FormRecognizer/src/RecognizedForm.cs

```diff
@@ -83,13 +85,21 @@ internal RecognizedForm(DocumentResult_internal documentResult, IReadOnlyList<Pa
             return fieldDictionary;
         }
 
-        private static IReadOnlyList<FormPage> ConvertPages(IReadOnlyList<PageResult_internal> pageResults, IReadOnlyList<ReadResult_internal> readResults)
+        private static IReadOnlyList<FormPage> ConvertUnsupervisedPages(PageResult_internal pageResult, IReadOnlyList<ReadResult_internal> readResults, int pageIndex)
```

Comment:
nit: Is it worth having this method?  You could do it inline...

Reply:
Good point. Updated it.